genesis/src/main.rs

```diff
@@ -25,22 +28,44 @@ use std::time::{Duration, Instant};
 
 pub const BOOTSTRAP_LEADER_LAMPORTS: u64 = 42;
 
+pub const VOTE_PROGRAM_NAME: &str = "vote_program";
+pub const STAKE_PROGRAM_NAME: &str = "stake_program";
+pub const SYSTEM_PROGRAM_NAME: &str = "system_program";
+
 pub enum AccountFileFormat {
     Pubkey,
     Keypair,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct PrimordialAccountDetails {
+    balance: u64,
+    owner: String,
```

Comment:
We want to be able to supply the account data as well

Reply:
@mvines What does account data include?

Reply:
Sure, this: https://github.com/solana-labs/solana/blob/5ffb6b874b5f5a294a3ac73d1e2783ce782e81b0/sdk/src/account.rs#L11

Being able to set the `executable` bit is also useful

Reply:
done